provision/acc_provision/templates/provision-config.yaml

```diff
@@ -146,7 +146,7 @@ registry:
   # enable_opflex_agent_reconnect: True                 # default is False, set it as True to enable opflex agent reconnect after vm migration
   # opflex_agent_statistics: False                      # default is True, set this to False if statistics load is very high
   # add_external_contract_to_default_epg: True          # override if needed, default is False
-
+  # disable_hpp_rendering: False                        # set to true to disable HPP rendering which disables k8s network policies functionality, default is false
```

Comment:
can we move this below L97 so as to have some logical grouping of the configurations?

Reply:
Done.